solutions/rcs-quickstart.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Step 1: Create 1 hub kind cluster and 2 managed Kind Clusters using a script from the OCM repository
+git clone https://github.com/open-cluster-management-io/OCM
+./OCM/solutions/setup-dev-environment/local-up.sh
+rm -rf OCM/
+
+# Step 2: Create the bin directory if it does not exist
+mkdir bin
+
+# Step 3: Install capp CRD on hub cluster 
+kubectl config use-context kind-hub
+git clone https://github.com/dana-team/container-app-operator capp
+make -C capp install
+
+# Step 4: Install and deploy capp and prerequisites on managed clusters
+kubectl config use-context kind-cluster1
+make -C capp/ prereq
+make -C capp deploy IMG=danateam/container-app-operator:release-0.1.3
+kubectl config use-context kind-cluster2
+make -C capp/ prereq
+make -C capp deploy IMG=danateam/container-app-operator:release-0.1.3
```

Comment:
Can we have the IMG be a variable which is called using the Makefile? We'd probably forget updating the releases here.